packages/core/helper-plugin/src/icons/RemoveRoundedButton.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 
-import { IconButton } from '@strapi/design-system';
+import { IconButton, IconButtonProps } from '@strapi/design-system';
 import { Minus } from '@strapi/icons';
-import styled from 'styled-components';
+import styled, { DefaultTheme } from 'styled-components';
 
 import { pxToRem } from '../utils/pxToRem';
 
 const StyledIconButton = styled(IconButton)(
-  ({ theme }) => `
+  ({ theme }: { theme: DefaultTheme }) => `
```

Comment:
I don't think you need this, the `.d.ts` file should augment the global scope of the package, was this a legacy?